docs/reference/setup-tools/kubeadm/kubeadm-join.md

```diff
@@ -21,6 +21,13 @@ This action consists of the following steps:
    authenticity of that data. The root CA can also be discovered directly via a
    file or URL.
 
+1. If kubeadm is invoked with `--feature-gates=DynamicKubeletConfig` enabled,
+   first kubeadm retrives the kubelet init configuration from the master and writes is to
+   the disk; then, as soon as kubelet starts, kubeadm updates the node `Node.spec.configSource`. 
```

Comment:
>the disk; then, as soon as kubelet starts, kubeadm updates the node `Node.spec.configSource`. 

the disk. When kubelet starts up, kubeadm updates the `spec.configSource` property of the node.